game/common/items/item.py

```diff
@@ -30,14 +40,47 @@ def value(self, value: int) -> None:
             raise ValueError(f'{self.__class__.__name__}.value must be an int.')
         self.__value = value
 
+    @quantity.setter
+    def quantity(self, quantity: int) -> int:
```

Comment:
Do setter properties implicitly return the old/new value? I think this should be None rather than int return type, but I'm not familiar enough with properties to definitively say that.